rust/agents/updater/src/updater.rs

```diff
@@ -107,7 +107,14 @@ impl NomadAgent for Updater {
             self.submitted_update_count.clone(),
         );
 
+        let fail_check = self.is_home_failed();
```

Comment:
nit: `let is_failed = self.is_home_failed();`, just think "is_failed" is more readable

Reply:
+1, but addressed below